packages/react-native/Libraries/AppDelegate/RCTAppDelegate.mm

```diff
@@ -107,7 +107,7 @@ - (void)_logWarnIfCreateRootViewWithBridgeIsOverridden
   if (currentClassImp != baseClassImp) {
     NSString *warnMessage =
         @"If you are using the `createRootViewWithBridge` to customize the root view appearence,"
-         "for example to set the backgroundColor, please migrate to `customiseView` method.\n"
+         "for example to set the backgroundColor, please migrate to `customizeRootView:` method.\n"
```

Comment:
looking at the TODO at line 101, we can probably remove this function! :D

Reply:
@cipolleschi Done~.